src/idl_gen_swift.cpp

```diff
@@ -0,0 +1,719 @@
+#include <unordered_set>
+
+#include "flatbuffers/code_generators.h"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/idl.h"
+#include "flatbuffers/util.h"
+
+namespace flatbuffers {
+
+namespace swift {
+
+inline std::string GenIndirect(std::string reading) {
+  return "{{ACCESS}}.indirect(" + reading + ")";
+}
+
+inline std::string GenArrayMainBody(std::string optional = "") {
```

Comment:
V813 Decreased performance. The 'optional' argument should probably be rendered as a constant reference. idl_gen_swift.cpp 16